arbnode/transaction_streamer.go

```diff
@@ -1366,6 +1402,12 @@ func (s *TransactionStreamer) pollSubmittedTransactionForFinality(ctx context.Co
 		log.Warn("failed to set the submitted hash to nil", "err", err)
 		return s.config().EspressoTxnsPollingInterval
 	}
+	lastConfirmedPos := submittedTxnPos[len(submittedTxnPos)-1]
```

Comment:
we should check here that len(submittedTxnPos) > 0 or else it will lead to a panic if submittedTxnPos[0-1]